integration/tests/cook/util.py

```diff
@@ -318,13 +318,10 @@ def retrieve_mesos_url(varname='MESOS_PORT', value='5050'):
     mesos_url = os.getenv('COOK_MESOS_LEADER_URL')
     if mesos_url is None:
         mesos_port = os.getenv(varname, value)
-        cook_url = retrieve_cook_url()
-        _wait_for_cook(cook_url)
-        mesos_master_hosts = settings(cook_url).get('mesos-master-hosts', ['localhost'])
-        resp = session.get('http://%s:%s/redirect' % (mesos_master_hosts[0], mesos_port), allow_redirects=False)
+        resp = session.get(f'http://localhost:{mesos_port}/redirect', allow_redirects=False)
```

Comment:
Why is it ok to assume localhost here? I'm imagining a case where you run the integration tests on a production cluster and then the tests that check the number of hosts. What am I missing?

Reply:
You are correct, this is only a decent assumption when running locally. For other environments, we will have to set the `COOK_MESOS_LEADER_URL` environment variable (see 3 lines above).

Reply:
ah ha! That is what I was missing. Thanks, I knew it was something stupid on my part.